pages/data.py

```diff
@@ -58,6 +59,11 @@ def render_content(tab, store_uuids, store_trips):
             col['label'] for col in perm_utils.get_allowed_named_trip_columns()
         )
         has_perm = perm_utils.has_permission('data_trips')
+    elif tab == 'tab-demographics-datatable':
+        data = store_demographics["data"]
+        columns = list(data[0].keys())
+        has_perm = perm_utils.has_permission('data_demographics')
```

Comment:
why are we not supporting excluded columns here? Please see my previous comment
https://github.com/e-mission/op-admin-dashboard/pull/63#discussion_r1313033861

>  You should exclude the entries that the config wants to be excluded (currently marked as TODO) and potentially exclude several of the entries here. But we should not have a list of valid cols

I don't see where you are excluding the entries that are marked as needing exclusion in the config.

Reply:
For backwards compatibility, if the config doesn't have the `data_demographic_columns_exclude`, then you can set it to a blank array. add a note to remove the backwards compat change after a couple of years when all the current studies are done.